plextraktsync/commands/watch.py

```diff
@@ -120,6 +120,9 @@ def on_play(self, event: PlaySessionStateNotification):
 
         movie = m.plex.item
         percent = m.plex.watch_progress(event.view_offset)
+        
+        if percent >= self.scrobblers.threshold:
```

Comment:
```suggestion
        if percent >= self.threshold:
```

Reply:
I guess both will work but sure `self.treshold` is fine :)

Reply:
you have not tested your changes with f12ccd2a7e36ab4cc11e6ea0e745d0dad9f9fb91 commit, the self.threshold is not initialized in this branch like the other pr you took changes from.